config_test.go

```diff
@@ -386,3 +386,64 @@ func TestCustomTag(t *testing.T) {
 	require.Equal(t, "name", s.Name)
 	require.Equal(t, 10, s.Age)
 }
+
+func TestSliceField(t *testing.T) {
+	t.Run("Empty slice", func(t *testing.T) {
+		s := struct {
+			Letters []string `config:"letters"`
+		}{}
+
+		st := store{
+			"letters": "a,b,c",
+		}
+		e := []string{
+			"a",
+			"b",
+			"c",
+		}
+		err := confita.NewLoader(st).Load(context.Background(), &s)
+		require.NoError(t, err)
+		require.EqualValues(t, e, s.Letters)
+	})
+
+	t.Run("Non-empty slice", func(t *testing.T) {
```

Comment:
Is this behaviour what we want? Aren't we introducing a "surprising" feature? For all other fields, we're "replacing" the content when loading stuff. Why would we just append here?